python/langsmith/_internal/_serde.py

```diff
@@ -120,13 +111,11 @@ def _elide_surrogates(s: bytes) -> bytes:
     return result
 
 
-def _dumps_json_single(
-    obj: Any, default: Optional[Callable[[Any], Any]] = None
```

Comment:
Removed the `default` parameter as `_dump_json_single` is only ever called with `_serialize_json` and falling back to `_simple_default` was confusing